servers/lib/src/cloudcmd/cloudcmd.ts

```diff
@@ -0,0 +1,31 @@
+import { INestApplication } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Server } from 'socket.io';
+import * as cloudcmd from 'cloudcmd';
+
+const runCloudCMD = (app: INestApplication, optionsPath: string) => {
```

Comment:
If LOCAL_PATH is passed as function argument, then the ConfigService is no longer needed.

Reply:
Yes, that is correct however, since we already needed the NestApplication I thought I would spare the function another argument, by just getting the ConfigService.

I will change this if you want LOCAL_PATH as a function argument

Reply:
Please pass LOCAL_PATH as function argument.